pkg/storage/credential_store.go

```diff
@@ -1,32 +1,43 @@
-package credentials
+package storage
 
 import (
 	"context"
 	"fmt"
 	"strings"
 
 	"get.porter.sh/porter/pkg/secrets"
-	"get.porter.sh/porter/pkg/storage"
 	"get.porter.sh/porter/pkg/tracing"
 	"github.com/cnabio/cnab-go/secrets/host"
 	"github.com/hashicorp/go-multierror"
 	"github.com/pkg/errors"
 )
 
-var _ Provider = &CredentialStore{}
+var _ CredentialSetProvider = &CredentialStore{}
 
 const (
 	CollectionCredentials = "credentials"
 )
 
-// CredentialStore provides access to credential sets by instantiating plugins that
-// implement CRUD storage.
+// CredentialStore provides access to work with Porter
+// credential sets, usually referred to as "credentials" as a shorthand.
+//
+// Credential Sets define mappings from a credential needed by a bundle to where
```

Comment:
I wonder if this documentation applies better with the `CredentialSet` struct instead of here